processing/src/test/java/org/apache/druid/query/rowsandcols/semantic/CombinedSemanticInterfacesTest.java

```diff
@@ -54,6 +61,18 @@ public CombinedSemanticInterfacesTest(
     super(name, fn);
   }
 
+  @Test
+  public void testColumnSelectorFactoryNonExistentColumn()
+  {
+    RowsAndColumns rac = make(MapOfColumnsRowsAndColumns.fromMap(
+        ImmutableMap.of(
+            "some", new IntArrayColumn(new int[] {3, 54, 21, 1, 5, 54, 2, 3, 92}))));
+    AtomicInteger currRow = new AtomicInteger();
+    ColumnSelectorFactory csfm = ColumnSelectorFactoryMaker.fromRAC(rac).make(currRow);
+    DruidException e = assertThrows(DruidException.class, () -> csfm.makeColumnValueSelector("nonexistent"));
+    assertThat(e, DruidExceptionMatcher.defensive().expectMessageIs("Column[nonexistent] not found!"));
+  }
+
```

Comment:
This should test for an all-null column of the same length, not for an exception.